library/src/main/kotlin/me/thanel/swipeactionview/SwipeActionView.kt

```diff
@@ -253,6 +250,10 @@ class SwipeActionView : FrameLayout {
             field = newResistance
         }
 
+    var resetDelay = 0L
```

Comment:
This should be set to 200 to preserve the behavior from the current version.

Also, could you add a documentation comment describing this property?

Reply:
Sure!